app/database/operator/server_data_operator/transformers/post.ts

```diff
@@ -41,7 +41,12 @@ export const transformPostRecord = ({action, database, value}: TransformerArgs):
         post.updateAt = raw.update_at;
         post.isPinned = Boolean(raw.is_pinned);
         post.message = raw.message;
-        post.metadata = raw.metadata && Object.keys(raw.metadata).length ? raw.metadata : null;
+
+        // When we extract the posts from the threads, we don't get the metadata
+        // So, it might not be present in the raw post, so we use the one from the record
```

Comment:
This seems error prone. Shouldn't this be solved at server level?

Reply:
@larkox we don't return post metadata in the thread APIs, 
FYI: metadata holds embeds, emojis, files, images, reactions, acknowledgements and priority.

The reason is to keep the thread APIs the lighter as possible